hint.cabal

```diff
@@ -45,14 +45,18 @@ test-suite unit-tests
 
 library
   build-depends: base == 4.*,
-                 ghc >= 7.10 && < 8.4,
+                 ghc >= 7.10 && < 8.5,
```

Comment:
Is <8.5 better than 8.6 here? I'm not very familiar with how GHC numbers releases.